src/settings/App.tsx

```diff
@@ -205,40 +206,38 @@ export class App extends Component<unknown, AppState> implements OC.Plugin<OC.Se
 	render() {
 		const isCirclesEnabled = loadState('groupfolders', 'isCirclesEnabled', false)
 		const groupHeader = isCirclesEnabled
-			? t('groupfolders', 'Group or team')
+			? t('groupfolders', 'Group or Team')
```

Comment:
not sure about this is it a name?

Reply:
Kind of for the circles replacement? It's kind of a known item now. "Circles", "Teams". :thinking: 
Not sure :)